SpatialGDK/Source/SpatialGDK/Public/SpatialView/ViewCoordinator.h

```diff
@@ -49,34 +49,47 @@ class ViewCoordinator
+	virtual const EntityView& GetView() const override;
 
-	const ComponentData* GetComponent(Worker_EntityId EntityId, Worker_ComponentId ComponentId) const;
+	virtual const FString& GetWorkerId() const override;
 
-	void SendAddComponent(Worker_EntityId EntityId, ComponentData Data, const FSpatialGDKSpanId& SpanId);
-	void SendComponentUpdate(Worker_EntityId EntityId, ComponentUpdate Update, const FSpatialGDKSpanId& SpanId);
-	void SendRemoveComponent(Worker_EntityId EntityId, Worker_ComponentId ComponentId, const FSpatialGDKSpanId& SpanId);
+	virtual Worker_EntityId GetWorkerSystemEntityId() const override;
+
+	virtual const ComponentData* GetComponent(Worker_EntityId EntityId, Worker_ComponentId ComponentId) const override;
+
+	virtual void SendAddComponent(Worker_EntityId EntityId, ComponentData Data, const FSpatialGDKSpanId& SpanId = {}) override;
+	virtual void SendComponentUpdate(Worker_EntityId EntityId, ComponentUpdate Update, const FSpatialGDKSpanId& SpanId = {}) override;
+	virtual void SendRemoveComponent(Worker_EntityId EntityId, Worker_ComponentId ComponentId,
+									 const FSpatialGDKSpanId& SpanId = {}) override;
+	virtual void SendEntityCommandResponse(Worker_RequestId RequestId, CommandResponse Response,
+										   const FSpatialGDKSpanId& SpanId = {}) override;
+	virtual void SendEntityCommandFailure(Worker_RequestId RequestId, FString Message, const FSpatialGDKSpanId& SpanId = {}) override;
+	virtual void SendMetrics(SpatialMetrics Metrics) override;
+	virtual void SendLogMessage(Worker_LogLevel Level, const FName& LoggerName, FString Message) override;
+
+	virtual Worker_RequestId SendReserveEntityIdsRequest(uint32 NumberOfEntityIds, FRetryData RetryData) override;
+	virtual Worker_RequestId SendCreateEntityRequest(TArray<ComponentData> EntityComponents, TOptional<Worker_EntityId> EntityId,
+													 FRetryData RetryData, const FSpatialGDKSpanId& SpanId = {}) override;
+	virtual Worker_RequestId SendDeleteEntityRequest(Worker_EntityId EntityId, FRetryData RetryData,
+													 const FSpatialGDKSpanId& SpanId = {}) override;
+	virtual Worker_RequestId SendEntityQueryRequest(EntityQuery Query, FRetryData RetryData) override;
+	virtual Worker_RequestId SendEntityCommandRequest(Worker_EntityId EntityId, CommandRequest Request, FRetryData RetryData,
+													  const FSpatialGDKSpanId& SpanId = {}) override;
+
+	virtual bool HasEntity(Worker_EntityId EntityId) const override;
+	virtual bool HasComponent(Worker_EntityId EntityId, Worker_ComponentId ComponentId) const override;
+	virtual bool HasAuthority(Worker_EntityId EntityId, Worker_ComponentSetId ComponentSetId) const override;
+
+	Worker_RequestId SendEntityCommandRequest(Worker_EntityId EntityId, CommandRequest Request, TOptional<uint32> TimeoutMillis = {},
+											  const FSpatialGDKSpanId& SpanId = {});
 	Worker_RequestId SendReserveEntityIdsRequest(uint32 NumberOfEntityIds, TOptional<uint32> TimeoutMillis = {});
```

Comment:
Are these still needed? They look like a duplicate of the overriden ones.

Reply:
They aren't quite as they don't have retry data and skip that process, however this isn't obvious from the api and I can't think where we'd use them. So maybe we just delete them.